pages/index.tsx

```diff
@@ -8,6 +12,137 @@ const Home = () => {
         <title>Kedro</title>
       </Head>
       <Hero />
+      <section className={style.features}>
+        <h3 className={style.sectionTitle}>Features</h3>
+        <FeatureDetails
+          buttonLink="https://demo.kedro.org/"
+          buttonText="Explore Live Demo"
+          iframeLink="https://demo.kedro.org/"
+          subtitle={
+            <>
+              <a
+                href="https://github.com/kedro-org/kedro-viz"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                Kedro&#39;s pipeline visualisation plugin
+              </a>{' '}
```

Comment:
minor: it seems that the `{' '}` empty space is applied by prettier, but it might be more indicative to use the commonly used  `&nbsp` to help indicate the intent of spacing, moreover it being easier to spot the number of white spaces applied.

Reply:
It is a bit odd. With the non-breaking space here we'd have to put it in an HTML element, like this `<span>&nbsp;</span>`, otherwise it'll just render **$nbsp** as text. That doesn't feel like much of an improvement. What do you think?